sdk/search/search-documents/src/generated/data/models/index.ts

```diff
@@ -107,7 +108,7 @@ export interface SearchRequest {
   orderBy?: string;
   /**
    * A value that specifies the syntax of the search query. The default is 'simple'. Use 'full' if
-   * your query uses the Lucene query syntax. Possible values include: 'simple', 'full'
+   * your query uses the Lucene query syntax. Possible values include: 'Simple', 'Full'
```

Comment:
interesting that the comment uses a different casing

Reply:
Recently the swagger has undergone some changes as refactorings. All the changes to generated code is a result of that. Link: https://github.com/Azure/azure-rest-api-specs/pull/9301/files#diff-a01e10da9dba0149960db35a9d106976L2632

Reply:
This is the result of PR: https://github.com/Azure/azure-rest-api-specs/pull/9301/files#diff-a01e10da9dba0149960db35a9d106976L2632